pkg/scanners/helm/test/testdata/expected/mysql/templates/secrets.yaml

```diff
@@ -0,0 +1,15 @@
+# Source: mysql/templates/secrets.yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: mysql
+  namespace: 
+  labels:
+    app.kubernetes.io/name: mysql
+    helm.sh/chart: mysql-8.8.26
+    app.kubernetes.io/instance: mysql
+    app.kubernetes.io/managed-by: Helm
+type: Opaque
+data:
+  mysql-root-password: "aGZYYW1vN3V5NA=="
```

Comment:
Squealer is going to have a great time here :D

Reply:
Squealer Rocks People! :guitar: